code/modules/jobs/job_types/ncr.dm

```diff
@@ -324,7 +324,7 @@ Weapons		Service Rifle, Grease Gun, 9mm pistol, all good.
 
 /datum/outfit/loadout/sergeantrifleman
 	name = "Lead Rifleman"
-	suit_store = /obj/item/gun/ballistic/automatic/assault_carbine/worn
+	suit_store = /obj/item/gun/ballistic/automatic/assault_carbine
```

Comment:
Excuse me..?

Reply:
What's the problem with this now? The worn assault carbine has 18 damage and 14 spread if I recall, it's garbage.